### PR TITLE
Bump jQuery 3.5.0 to 3.5.1

### PR DESCRIPTION
**Pull request by cpsievert**
This bumps jQuery 3.5.0 -> 3.5.1 and also rolls back the BS3 patch done in #2897 (since the problem (#2896) can be fixed by just bumping jQuery).

### CODE REVIEW

File: tools/updateAndPatchBootstrap.R [hunk 1]
```diff
@@ -1,38 +0,0 @@
-version <- "3.4.1"
```
Comment (wch) [inline tools/updateAndPatchBootstrap.R:1]: Would it make sense to keep this script, just without the patch stuff?

Comment (cpsievert) [reply tools/updateAndPatchBootstrap.R:1]: I guess, but hopefully in the near future we'll be using `bootstraplib::bootstrap()` ;)

**Comment by wch** [comment]
Can you also add a script for updating jQuery? We'll be doing this again in the future and it would be nice to have a script for it.

File: tools/updateAndPatchBootstrap.R (hunk 1 above)
Comment (cpsievert) [reply tools/updateAndPatchBootstrap.R:1]: (Also, even if we don't, I doubt it'll be that useful as BS3 is end-of-life)

**Pull request merged**